### DISCUSSION TITLE
Sunbird RC Roadmap: Community Huddle on Alternate Thursdays at 3 PM IST

### DISCUSSION THREAD

**Discussion by surendrasinghs** [Sunbird-RC/community#445] [General]
Hi all!

Excited to announce Sunbird RC Roadmap Session where the community can come together to shape the future of RC collectively.

Here's the draft roadmap that is evolving and will be used to anchor the call: https://github.com/orgs/Sunbird-RC/projects/16

Contributions, ideas, suggestions are welcome! Please [create a discussion](https://github.com/orgs/Sunbird-RC/discussions) or [raise a ticket](https://github.com/Sunbird-RC/community/issues)

Join us on alternate Thursdays from 3 to 4 PM IST [using this link](https://meet.google.com/jmx-zbxv-oui
)

Use the button to add this to your <a target="_blank" href="https://calendar.google.com/calendar/event?action=TEMPLATE&amp;tmeid=NzlkNDRhZjQzM2RsdGgwYTMxc2lkdDhlM2xfMjAyMzAzMjNUMDkzMDAwWiBzdXJlbmRyYXNpbmdoQGVrc3RlcC5vcmc&amp;tmsrc=surendrasingh%40ekstep.org&amp;scp=ALL"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en-GB.gif"></a>

See you this Thursday 23rd March at 3 PM!

**Comment by gudipatiharitha** [comment]
Hi - I waited for the call for 5 minutes and dropped off , Could you please let me know if this is still happening at the same link

**Reply by parthlawate** [reply]
@gudipatiharitha Today the call is moved to 4 PM
